boards/waspmote-pro/include/board.h

```diff
@@ -180,6 +181,27 @@ extern "C" {
 #define CONFIG_ZTIMER_USEC_WIDTH    (16)
 /** @} */
 
+/**
+ * @name Onboard micro-sd slot pin definitions
+ * @{
+ */
+#define SDCARD_SPI_PARAM_SPI        SPI_DEV(0)
```

Comment:
```suggestion
#define SDCARD_SPI_PARAM_SPI        SPI_DEV(0) /**< SPI bus used */
``

I know this is annoying, but CI is now enforcing those to be documented :confused:

Reply:
Now tests/mtd_raw works:

```
$ BOARD=waspmote-pro PORT=/dev/ttyUSB0 make -C tests/mtd_raw all flash term
[...]
2022-08-12 13:35:11,968 # main(): This is RIOT! (Version: 2022.10-devel-291-g4edab1-waspmote-sdcard)
2022-08-12 13:35:11,984 # Manual MTD test
2022-08-12 13:35:12,145 # init MTD_0… OK (1915 MiB)
> info
2022-08-12 13:35:17,631 # info
2022-08-12 13:35:17,647 # mtd devices: 1
2022-08-12 13:35:17,663 #  -=[ MTD_0 ]=-
2022-08-12 13:35:17,679 # sectors: 3921920
2022-08-12 13:35:17,695 # pages per sector: 1
2022-08-12 13:35:17,711 # page size: 512
2022-08-12 13:35:17,727 # total: 1915 MiB
```

But tests/vfs_default does not:

```
BOARD=waspmote-pro PORT=/dev/ttyUSB0 make -C tests/vfs_default all flash term
[...]
2022-08-12 13:51:18,714 # main(): This is RIOT! (Version: 2022.10-devel-293-g8782e-waspmote-sdcard)
2022-08-12 13:51:18,730 # mount points:
2022-08-12 13:51:18,730 # 
2022-08-12 13:51:18,746 # data dir: /sd0
> ls /sd0
2022-08-12 13:51:52,706 # ls /sd0
2022-08-12 13:51:52,723 # vfs_opendir error: -ENOENT
> ls /
2022-08-12 13:51:54,082 # ls /
2022-08-12 13:51:54,114 # vfs_opendir error: -ENOENT
```

Reply:
oh, the comment above should be a general comment...
(defines documented in the latest commit)